pkg/storage/s3_test.go

```
@@ -452,6 +452,13 @@ func (r *testStorageSuite) TestS3Others(c *C) {
 	defineS3Flags(&pflag.FlagSet{})
 }
 
+func (r *testStorageSuite) TestS3URI(c *C) {
+	backend, err := ParseBackend("s3://bucket/prefix/", nil)
```

This error is not checked.